ext/bg/js/backend.js

```diff
@@ -804,6 +788,76 @@ class Backend {
 
     // Utilities
 
+    _getBrowserIconTitle() {
+        return (
+            chrome.browserAction !== null &&
+            typeof chrome.browserAction === 'object' &&
+            typeof chrome.browserAction.getTitle === 'function' ?
+            new Promise((resolve) => chrome.browserAction.getTitle({}, resolve)) :
+            Promise.resolve('')
+        );
```

Comment:
The ternary should probably be indented for the parts after `?` or the condition could be stored in a constant.

Reply:
0628c333d9b0fc15aeb2ec8cdead9a8dff4fb087